scopes/component/snapping/snap-cmd.ts

```diff
@@ -12,44 +12,52 @@ import { BasicTagSnapParams } from './tag-model-component';
+  description = 'create an immutable and exportable component snapshot (non-release version)';
   extendedDescription: string;
   group = 'development';
   arguments = [
     {
       name: 'component-pattern',
-      description: `${COMPONENT_PATTERN_HELP}. By default, all new and modified components are snapped.`,
+      description: `${COMPONENT_PATTERN_HELP}. By default, only new and modified components are snapped (add --unmodified to snap all components in the workspace).`,
     },
   ];
   helpUrl = 'docs/components/snaps';
   alias = '';
   options = [
-    ['m', 'message <message>', 'log message describing the latest changes'],
+    ['m', 'message <message>', 'snap message describing the latest changes - will appear in component history log'],
     ['', 'unmodified', 'include unmodified components (by default, only new and modified components are snapped)'],
     ['', 'unmerged', 'complete a merge process by snapping the unmerged components'],
-    ['b', 'build', 'not needed for now. run the build pipeline in case the feature-flag build-on-ci is enabled'],
+    [
+      'b',
+      'build',
+      'not needed for now. run the build pipeline locally in case the feature-flag build-on-ci is enabled',
+    ],
     [
       '',
       'editor [editor]',
-      'open an editor to write a tag message for each component. optionally, specify the editor-name (defaults to vim).',
+      'open an editor to write a snap message per component. optionally specify the editor-name (defaults to vim).',
     ],
     ['', 'skip-tests', 'skip running component tests during snap process'],
     ['', 'skip-auto-snap', 'skip auto snapping dependents'],
-    ['', 'disable-snap-pipeline', 'skip the snap pipeline'],
+    [
+      '',
+      'disable-snap-pipeline',
+      'skip the snap pipeline. this will for instance skip packing and publishing component version for install, and app deployment',
+    ],
     ['', 'force-deploy', 'DEPRECATED. use --ignore-build-error instead'],
-    ['', 'ignore-build-errors', 'run the snap pipeline although the build pipeline failed'],
+    ['', 'ignore-build-errors', 'proceed to snap pipeline even when build pipeline fails'],
     [
       'i',
       'ignore-issues [issues]',
       `ignore component issues (shown in "bit status" as "issues found"), issues to ignore:
 [${Object.keys(IssuesClasses).join(', ')}]
 to ignore multiple issues, separate them by a comma and wrap with quotes. to ignore all issues, specify "*".`,
     ],
-    ['a', 'all', 'DEPRECATED (not needed anymore, it is the default now). snap all new and modified components'],
+    ['a', 'all', 'DEPRECATED (not needed anymore, now the default). snap all new and modified components'],
     [
       '',
       'fail-fast',
-      'stop pipeline execution on the first failed task (by default a task is skipped only when its dependent failed)',
+      'stop pipeline execution on the first failed task (by default a task is skipped only when its dependency failed)',
```

Comment:
It's actually "dependent".
For example, the tester task depends on the compiler task.
By default, the tester task is skipped only when the compiler task failed. 
The compiler is the dependent of the tester.

Reply:
Not how i understand it.
A dependent is something that depends on me, not what i depend on... (your kids are your dependents, your their dependency :) 
If the tester task depends on the compiler then the compiler is its dependency, not its dependent...

Reply:
sorry, my bad, of course.